Source/common/CoderMacros.h

```diff
@@ -15,46 +15,53 @@
 #ifndef SANTA__COMMON__CODERMACROS_H
 #define SANTA__COMMON__CODERMACROS_H
 
-// Encode the property keyed by the property name.
+/// Encode the property keyed by the property name.
```

Comment:
What is up with the triple `/` for code comments?

Reply:
The hope was that they would be picked up as doc strings, but it didn't really work for me. Might be a macro-thing? Left them in cause using doc strings still felt right, but we're not at all consistent across the project...

Reply:
I originally used triple-slashes for docstrings because Xcode had some rudimentary support for them (which is why there are also `@return` and `@param` comments in a few places). Once we moved away from Xcode I stopped being picky about using them, though I agree that triple-slashes for docstrings to differentiate from "normal" comments is a good idea and we should do continue to do it.